src/BranchManager.php

```diff
@@ -69,7 +71,20 @@ public function getReleaseData($repositoryName)
             $devLastCommitSha
         );
 
-        $openPullRequests = $this->client->api('pull_request')->all('prestashop', $repositoryName, array('state' => 'open', 'base' => $usedBranch));
+        // Do not count dependabot commits since last release.
+        $dependabotCommit = 0;
+        if ($releaseDate != 'NA') {
+            $commits = $this->client->api('repo')->commits()->all('prestashop', $repositoryName, array('sha' => $devBranchUsed, 'since' => $release['created_at']));
+
+            foreach ($commits as $commit) {
+                $pos = strpos($commit['commit']['message'], 'dependabot');
```

Comment:
You "detect" dependabot commits by checking if a given commit message contains "dependabot" in it? 😅 

It is maybe not the most reliable way to detect it. Isn't it possible to look for the PR author and check if it is dependabot?

Reply:
Yes, I couldn't find any other way, so I will use the search api but there is a rate limit, I will add a sleep.